src/components/Categories.js

```diff
@@ -0,0 +1,11 @@
+function Categories() {
```

Comment:
- [ ] As required, I kindly suggest that you have a `<Button>` on your Categories page with the text "Check status"

Reply:
Hi @topeogunleye, 
Thank you